libs/SalesforceSDK/src/com/salesforce/androidsdk/auth/HttpAccess.java

```diff
@@ -149,50 +138,21 @@ public synchronized OkHttpClient getOkHttpClient() {
     }
 
     /**
-     * Detects network changes and sets the network connectivity status.
-     *
-     * @param context The context of the request.
-     * @param intent Intent.
-     */
-    @Override
-    public void onReceive(Context context, Intent intent) {
-        if (conMgr == null) {
-            return;
-        }
-
-        // Checks if an active network is available.
-        final NetworkInfo activeInfo = conMgr.getActiveNetworkInfo();
```

Comment:
We don't need the conditionals below or checking per type. Current type is all we need to establish if we're connected.